pkg/commands/refresh_test.go

```diff
@@ -0,0 +1,73 @@
+package commands
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestCheckRefreshInterval(t *testing.T) {
```

Comment:
please don't use ginkgo for pure unit-test. It's hard to maintain. We prefer ginkgo only when the test need K8s environment.

We prefer table driven test for normal unit-test, you can refer to https://dave.cheney.net/2019/05/07/prefer-table-driven-tests

Reply:
got it.